lte/gateway/python/setup.py

```diff
@@ -127,6 +127,7 @@
         'ovs>=2.13',
         'prometheus-client>=0.3.1',
         'aioeventlet==0.5.1',  # aioeventlet-build.sh
+        'parameterized==0.8.1',
```

Comment:
are we only using this dep for testing purposes? if so is there an alternative without installing this? we should limit the amount of deps being installed for the gateway

Reply:
Fixed.